app/models/spotify_playlist.rb

```diff
@@ -16,7 +16,7 @@
 # Indexes
 #
 #  index_spotify_playlists_on_account_id       (account_id)
-#  index_spotify_playlists_on_external_id      (external_id) UNIQUE
+#  index_spotify_playlists_on_external_id      (external_id)
```

Comment:
maybe use a double index? like on external_id and account_id. To ensure an account does not have a playlist more than once